schema/v1/app.cue

```diff
@@ -92,6 +92,7 @@ package v1
 	ports:                          #PortSingle | *[...#Port] | #PortMap
 	[=~"probes|probe"]:             #Probes
 	[=~"depends[oO]n|depends_on"]:  string | *[...string]
+	[=~"mem|memory"]: 							int
```

Comment:
looks like everyhting else in this file is separated by spaces, not tabs. can you change this to match so it aligns properly in diffs?